src/table/components/FooterWrapper.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import Paper from '@mui/material/Paper';
+
+const Portal = ({ children, target }) => ReactDOM.createPortal(children, target);
+
+export default function FooterWrapper({ children, theme, footerContainer }) {
+  const paperTablePaginationStyle = {
+    height: 48,
+    display: 'flex',
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+    paddingRight: 1,
+    boxShadow: 'none',
+    borderStyle: 'solid',
+    borderWidth: '0px 0px 1px 0px',
+    borderRadius: 0,
+    borderColor: theme.table.pagination.borderColor,
+    color: theme.table.pagination.color,
+    backgroundColor: theme.table.backgroundColor,
+  };
+
+  return footerContainer ? (
+    <Portal target={footerContainer}>{children}</Portal>
```

Comment:
with the FooterWrapper component I think using ReactDOM.createPortal direct (without Portal component) works
(and looks nicer)
```suggestion
    ReactDOM.createPortal(children, footerContainer);
```

Reply:
nice, I'll try it once the qmfe is back up